README.md

```diff
@@ -15,11 +15,13 @@ using `babylonnet.sh` or `mainnet.sh` scripts.
 
 ## Build Instructions
 
-Run `nix-build -A tezos-client-static -o tezos-static`
-to build staticically linked `tezos-client` executable.
+Run `nix-build -A tezos-client-{mainnet, babylonnet} -o tezos-client`
+to build staticically linked `tezos-client-mainnet` or `tezos-client-babylonnet`
+executable.
 
-Run `nix-build -A tezos-client-{rpm, deb}-package -o tezos-client-package` in order
-to build native `.rpm` or `.deb` package for Red Hat and Debian-based distros.
+Run `nix-build -A {mainnet, babylonnet}-{rpm, deb}-package -o tezos-client-package --arg timestamp $(date -I | sed 's|-||g')` in order
```

Comment:
I think `$(date +"%Y%m%d")` would be better here since it's more portable (e.g. `-I` doesn't exist in Mac OS version of `date`) and explicit (and uses just one tool instead of two). Also, a more accurate timestamp, say down to minute precision `%Y%m%d%H%M`, might be better to allow multiple builds on the same day and treat have rpm/deb treat later builds as updates

Reply:
Same here, would be nice to have each command flavor fully spelled out and on a separate line - for easy copy-pasting of actual executable commands. Alternatively, a make file might be a good way to simplify invocation. E.g. `make rpm` could build  babylonnet rpm (client for current protocol version is probably of most interest which makes it a good default). Also, I wonder if rpm and deb builds can just use output of static build? that doesn't seem to be the case at the moment

Reply:
> I wonder if rpm and deb builds can just use the output of static build?
It actually uses the output of static build but in terms of nix derivation